wallet/net.go

```diff
@@ -0,0 +1,284 @@
+package wallet
+
+import (
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/any"
+	"github.com/textileio/textile-go/pb"
+	"golang.org/x/net/context"
+	"gx/ipfs/QmZoWKhxUmZ2seW4BzX6fJkNR8hh9PsGModr7q171yq2SS/go-libp2p-peer"
+	libp2pc "gx/ipfs/QmaPbCnUMBohSGo3KnxEa2bHqyJVVeEEcwtqJAYxerieBo/go-libp2p-crypto"
+	"gx/ipfs/QmcZfnkapfECQGcLZaf9B79NRg7cRa9EnZh4LSbkCzwNvY/go-cid"
+	"time"
+)
+
+//var OfflineMessageWaitGroup sync.WaitGroup
+
+func (w *Wallet) sendMessage(message *pb.Message, peerId string) error {
```

Comment:
wrapper around service, so that the wallet can directly call things like `sendMessage`